src/screens/Students.js

```diff
@@ -193,80 +205,104 @@ class Students extends Component {
 
               <ExportButton onClick={this.handleExportClick}>
-                <div/>
+                <div />
                 EXPORT STUDENT DATA
-                <ExportIcon/>
+                <ExportIcon />
               </ExportButton>
             </StudentListContainer>
 
-            <StyledDivider/>
+            <StyledDivider />
 
             <Route path={`/${schoolYearId}/students/new`} render={props => {
-              if(schoolYear) {
+              if (schoolYear) {
                 return (
                   <SubRouteWrapper>
-                    <CreateStudentForm schoolYear={schoolYear}/>
+                    <CreateStudentForm schoolYear={schoolYear} />
                   </SubRouteWrapper>
                 );
               } else {
                 return this.renderLoadingSubroute();
               }
-            }}/>
+            }} />
 
             <Route path={`/${schoolYearId}/students/edit/:studentId`} render={props => {
-              if(!schoolYear) return this.renderLoadingSubroute();
+              if (!schoolYear) return this.renderLoadingSubroute();
               const student = filteredStudents.find(student => student.id === +props.match.params.studentId);
 
-              if(!student) return (<Redirect to={`/${schoolYearId}/students`}/>);
+              if (!student) return (<Redirect to={`/${schoolYearId}/students`} />);
 
               return (
                 <SubRouteWrapper>
-                  <EditStudentForm 
+                  <EditStudentForm
                     key={student.id}
                     student={student}
                     schoolYear={schoolYear}
                   />
                 </SubRouteWrapper>
               );
-            }}/>
+            }} />
 
             <Route path={`/${schoolYearId}/students/view/:studentId`} render={props => {
-              if(!schoolYear) return this.renderLoadingSubroute();
+              if (!schoolYear) return this.renderLoadingSubroute();
               const studentId = +props.match.params.studentId;
               const student = filteredStudents.find(student => student.id === studentId);
 
-              if(!student) return (<Redirect to={`/${schoolYearId}/students`}/>);
+              if (!student) return (<Redirect to={`/${schoolYearId}/students`} />);
 
               return (
                 <SubRouteWrapper>
-                  <StudentInfoView 
+                  <StudentInfoView
                     key={`${student.id}-${schoolYear.id}` /* Force re-mounting/re-loading on change */}
                     student={student}
                     schoolYear={schoolYear}
                   />
                 </SubRouteWrapper>
               );
-            }}/>
-            
+            }} />
+
+            <Route path={`/${schoolYearId}/students/multiple`} render={props => {
+              const { selectedStudents } = this
+
+              if (selectedStudents.length <= 1) {
+                if (selectedStudents.length === 1) {
```

Comment:
Do we care if there is one student selected when this "multiple student" functionality is active? I really think we should care about if the list is empty or not, and carry on if there is 1+ student in the list.

Reply:
From the design, it seems that we want to show the selected student's detail if there is only one student selected: https://projects.invisionapp.com/share/KDT61RTZR5J#/screens/375865786

Reply:
OIC.